htdocs/adminhistory.php

```diff
@@ -33,7 +32,7 @@
     $error = $translate->t('Cache not found', '', '', 0);
 } elseif ($cache_id > 0) {
     $showhistory = true;
-    $cache = new Cache($cache_id);
+    $cache = new cache($cache_id);
     $cache->setTplHistoryData(0);
```

Comment:
I got an email notification about a comment here, but cant't find it, maybe because I squashed the commit?

This is a bugfix - the class name currently is "cache" in caches.class.php and accidentally was changed to "Caches" here during refactoring.